here-naksha-storage-http/src/test/java/com/here/naksha/storage/http/PrepareResultTest.java

```diff
@@ -0,0 +1,30 @@
+package com.here.naksha.storage.http;
+
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import java.net.http.HttpHeaders;
+import java.net.http.HttpResponse;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+class PrepareResultTest {
+
+  @Test
```

Comment:
What does this test do? If we intend to do something here, then suggest putting some comments. Else, let's remove the test and file to avoid confusion.

Reply:
Sorry. Forgot to remove